pkg/agent/util/dump.go

```diff
@@ -0,0 +1,40 @@
+// Copyright 2020 Antrea Authors
```

Comment:
The util is used for general utility like iptables util, ipset util, ip route util while the code of dump are too business specific and impossible to be reused.
Maybe moving it to a separate package pkg/agent/support or something similar? what do you think? @jianjuns @antoninbas

Reply:
pkg/agent/dump or pkg/agent/support sound good to me.

But what about shared funcs in pkg/util/dump.go, like DumpNetworkPolicyResources? Should we move the some funcs to pkg/apiserver, and still keep util funcs in pkg/util/dump.go?

Reply:
Agreed with @tnqn.

I think we can have `pkg/agent/support` and `pkg/controller/support`?

As for the shared functions, maybe we don't need to unify? The only "duplicate" method really is `DumpNetworkPolicyResources`. Maybe we have one "copy" in `pkg/agent/support` and one copy in `pkg/controller/support`. The implementation relies on `dumpAntctlGet`. Maybe we can find a good shared location for `dumpAntctlGet` that both `pkg/agent/support` and `pkg/controller/support` can import. Or rewrite `dumpAntctlGet` in such a way that it makes sense to have it in `pkg/util` or `pkg/antctl`.

Reply:
Later we can have more cases to share code between controller and agent (actually we already have monitor and querier). I feel better not to duplicate code. Might be ok to put the shared code in pkg/util for now, or if you guys think better we can either do pkg/shared/support/ (and we move other shared pkgs to pkg/shared/ too), or pkg/support/?

Reply:
Sorry, I should have published this comment early, just found it is in draft:
Then should we follow the current way in which monitor and querier share code between agent and controller, i.e. having pkg/support? At least we are consistent for now and can re-organize them together in future if needed.

Reply:
`pkg/support` works for me

Reply:
Updated, PTAL.